.github/workflows/main.yml

```diff
@@ -743,7 +743,7 @@ jobs:
                                 cd $HOME/agents/node/vcxagent-core && npm run test:integration)'
 
   # TODO: Add tests of iOS build
-  publish-ios-wrapper:
+  build-ios-wrapper:
```

Comment:
These jobs build and upload - publish - an an artifact. I would revert these renames.